tools/travis_script.py

```diff
@@ -151,10 +152,8 @@ def build_iotjs(buildtype, args=[], env=[]):
             exec_docker(DOCKER_TIZENRT_OS_PATH, [
                         'make', 'IOTJS_ROOT_DIR=' + DOCKER_IOTJS_PATH,
                         'IOTJS_BUILD_OPTION='
-                        '--profile=test/profiles/tizenrt.profile',
```

Comment:
Any reason we should drop jerry-ext? It seems better to follow the above FIXME.

Reply:
I think it can be deleted because the `jerry-debugger` is not enabled by default, so `jerry-ext` is not needed after #1798

Reply:
(If we keep this change, then the comment must be deleted as well, because it is not valid after this patch)